x-pack/plugins/file_upload/server/telemetry/mappings.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
```

Comment:
I'm not really sure how to test ..

Reply:
There's still a global telemetry test covering it [here](https://github.com/elastic/kibana/blob/683c138798de8fdc65d915af7616ee21ab45df5b/x-pack/test/api_integration/apis/telemetry/telemetry_local.js#L89) that should fail if telemetry isn't being generated. Checking locally produces this result as expected: 

![image](https://user-images.githubusercontent.com/1728976/75831237-34b1a880-5d70-11ea-816a-64c1a4f56576.png)